src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -285,6 +360,83 @@ public Builder schema(@Nullable String schema) {
             return this;
         }
 
+        /**
+         * Configure ssl usage.
+         *
+         * @param ssl
+         * @return this {@link Builder}
+         */
+        public Builder ssl(boolean ssl) {
+            this.ssl = ssl;
+            return this;
+        }
+
+        /**
+         * Configure ssl cert.
+         *
+         * @param sslCert
+         * @return this {@link Builder}
+         */
+        public Builder sslCert(@Nullable String sslCert) {
```

Comment:
Nit: Let's document that this should be a path to the SSL certificate in PEM format.